docs/portal_config.md

```diff
@@ -129,7 +132,7 @@ Below is an example, with inline comments describing what each JSON block config
           "name": "Documentation"
         }
       ],
-      "useProfileDropdown": false // optional; enables expiremental profile UI; defaults false, may change in future releases
+      "useProfileDropdown": false // optional; enables experimental profile UI; defaults false, may change in future releases
```

Comment:
it's not experimental anymore, is it?

Reply:
well... it kind of still is, because only MIDRC is using this setup now 😅